Resources/translations/validators.hu.yml

```diff
@@ -1,25 +1,28 @@
 fos_user:
     username:
         already_used: 'A felhasználónév már foglalt.'
-        blank: 'Kérjük, adjon meg egy felhasználónevet.'
+        blank: 'A felhasználónév nem lehet üres.'
         short: 'A felhasználónév túl rövid.'
         long: 'A felhasználónév túl hosszú.'
     email:
         already_used: 'Az emailcím már használatban van.'
-        blank: 'Kérjük, adjon meg egy emailcímet.'
+        blank: 'Az emailcím nem lehet üres.'
```

Comment:
Szerintem is. Ez volt ezelőtt is, gondoltam nem zavart senkit, azért hagytam benne :)